src/commands.rs

```diff
@@ -402,7 +410,22 @@ fn print_status(ctx: &mut Context<'_>, device: &nitrokey::DeviceWrapper<'_>) ->
 
 /// Inquire the status of the nitrokey.
 pub fn status(ctx: &mut Context<'_>) -> anyhow::Result<()> {
-  with_device(ctx, |ctx, device| print_status(ctx, &device))
+  let device_infos = find_devices(&ctx.config)?;
```

Comment:
Is the order in which devices are returned by `nitrokey-rs` guaranteed to be stable between invocations?

Reply:
Not by nitrokey-rs.  The order is determined by libnitrokey, which orders by USB vendor ID, then by the order returned by hidapi’s `hid_enumerate` function.  hidapi’s documentation does not mention the order.